publish/malicious/BUILD.bazel

```diff
@@ -4,7 +4,5 @@ malicious_binary(
     name = "replica",
     testonly = True,
     binary = "//rs/replica:malicious_replica",
-    visibility = [
-        "//ic-os/guestos/envs/dev-malicious:__pkg__",
-    ],
+    visibility = ["//rs:ic-os-pkg"],
```

Comment:
This seems potentially dangerous since previously you could only use the malicious replica only in the malicious image but now you can use it in any IC-OS image.